tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/contact_form.py

```diff
@@ -154,7 +154,7 @@ def tap_update(self, return_details=True):
             return ContactDetails(self.marionette)
         else:
             # else we drop back to the underlying app
-            Wait(self.marionette).until(lambda m: self.apps.displayed_app.name != self.name)
+            self.wait_to_not_be_displayed()
```

Comment:
I don't think this works here. This inherits from ContactForm which inherits from Base.
This test wants to check that the contacts app is not displayed anymore, so I think this has to be:
Contacts(self.marionette).wait_to_not_be_displayed()
You also need to import Contacts then, beforehand.